tests/gpu-container

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 set -eu
 
+# testflinger_queue: rockman
```

Comment:
This is fine for now, but eventually we'll want to move this out of the script itself and into the configuration file that schedules the run.

Reply:
Indeed, that's the plan.